sympy/solvers/ode/tests/test_single.py

```diff
@@ -333,6 +334,12 @@ def test_SingleODESolver():
     problem = SingleODEProblem(f(x).diff(x,4) + f(x).diff(x,2) - f(x).diff(x,3), f(x), x)
     assert problem.order == 4
 
+    problem = SingleODEProblem(f(x).diff(x, 3) + f(x).diff(x, 2) - f(x)**2, f(x), x)
+    assert problem.is_autonomous()
```

Comment:
I would `assert ... is True`

Generally if you want to test `f` then `assert f()` is not a good test because almost every possible object in Python is truthy (e.g. `assert [1, 2]`).